### ISSUE TITLE
pack_rows miss group / sub group associations

### ISSUE THREAD

**Issue by gmeurice** [haozhu233/kableExtra#443]
**Describe the bug**

in the following, one expect to have on element in Sub3.2. Moreover, there is some indentations issues.  I guess there is missing </tr> in the output of pack_rows

```
# code to reproduce the bug.
kable(mtcars[1:20, 1:6], caption = "Group Rows") %>%
 kable_styling("striped", full_width = F) %>%
 pack_rows(index = c("GR1" = 3, "GR2" = 7, "GR3" = 4, "GR4" = 6)) %>%
 pack_rows(index = c("Sub1.1" = 3, "Sub2.1" = 2, "Sub2.2" = 5, "Sub3.1" = 3, "Sub3.2" = 1, "Sub4.1" = 6), label_row_css = "color: #4d88ff;font-style:italic") 
```

**Comment by Markouski** [comment]
I would add that the issue persists only for 'html_document'.
The above code works fine when rendering 'pdf_document.'

**Comment by jokorn** [comment]
I think this is the same issue as #397.

**Comment by haozhu233** [comment]
@gmeurice @DzimitryM 

@jokorn kindly fixed this bug in #440. It was my bad for the delay. I realized this issue as I'm going through the backlog recently. I will update this package on CRAN in a few days.
